minio/console
# Added missing validations in add tenant modal

## What does this do?

Added missing validations in add tenant modal

## How to test?

- Open add tenant modal
- Verify that required fields ar correctly validated
